Sources/SessionManager/SessionManager.swift

```diff
@@ -46,41 +46,40 @@ public class SessionManager {
     }
 
     private func generateRandomSessionID() -> String? {
-        if let val = generatePrivateKeyData()?.toHexString().padStart(toLength: 64, padString: "0") {
+        if let val = generateRandomData(length: 32)?.toHexString().padStart(toLength: 64, padString: "0") {
```

Comment:
sessionid is actually a private key used for encryption/decryption.
pls keep generatePrivateKeyData

Reply:
Reverted.